### DISCUSSION TITLE
Nextcloud Sync Issue with Files Over 1MB on Proxmox Alpine Linux Installation

### DISCUSSION THREAD

**Discussion by Smile2Grow** [tteck/Proxmox#2531] [Help Needed]
I've been using Nextcloud Alpine bash script. Everything is set up and running smoothly, except for one significant issue that I'm facing with the Nextcloud instance.

It seems that any file over 1MB in size is not syncing correctly with my server. I've already adjusted the Apache php.ini configuration to allow for larger upload sizes well beyond 1MB, but this hasn't resolved the issue.

Has anyone else encountered this specific problem? If so, were you able to find a solution? I'm particularly interested in any configuration changes or tweaks within Proxmox or Nextcloud that might have been effective in overcoming this hurdle.

I appreciate any insights or suggestions you might have. Thank you in advance for your help!

**Comment by kilianyp** [comment]
I faced the same issue and could resolve it by setting `client_max_body_size 512m;` in `/etc/nginx/nginx.conf` (line 43).
